src/services/networkService.js

```diff
@@ -376,22 +376,43 @@ class NetworkService {
     }
   }
 
+  async getConnectedLedgerAddress () {
+    const transport = await Transport.create();
+    const eth = new Eth(transport);
+
+    let result;
+    for (let i = 0; i < 10; i++) {
+      const derivationPath = `44'/60'/${i}'/0/0`;
+      const { address } = await eth.getAddress(derivationPath);
```

Comment:
maybe it's faster to request 10 derivation paths simultaneously using `Promise.all`, then find the account later?

Reply:
yea tried this but ledger can only do one at a time :(